cms/templates/widgets/header.html

```diff
@@ -165,7 +165,7 @@ <h3 class="title"><span class="label"><span class="label-prefix sr">${_("Library
 
     <div class="wrapper wrapper-r">
       % if user.is_authenticated():
-      <nav class="nav-account nav-is-signedin nav-dd ui-right">
+      <nav class="nav-account nav-is-signedin nav-dd ui-right" aria-label="${_('Account')}">
         <h2 class="sr">${_("Help &amp; Account Navigation")}</h2>
```

Comment:
Since you're in this file, lets make this consistent with the aria-label above and just make it "Account Navigation"